.github/workflows/ci.yaml

```diff
@@ -46,3 +48,12 @@ jobs:
       - run: make lint
       - run: make test
       - run: make build
+
+      - uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'config'
+          ignore-unfixed: true
+          format: table
+          severity: 'CRITICAL,HIGH'
+          skip-dirs: vendor,node_modules
```

Comment:
Instead of ignoring `node_modules`, could we just _remove_ the `node_modules` directory after running `rake assets:precompile` in the Dockerfile?

Reply:
As per my research, it looks like `node_modules` is required at runtime. I’ll leave it as is, for now.